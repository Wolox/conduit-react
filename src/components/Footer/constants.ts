export const urlWolox = 'https://www.wolox.com.ar/';
