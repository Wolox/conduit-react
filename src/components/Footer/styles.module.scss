@import 'scss/variables/_colors';

.year{
    color: $white;

    .link{
        color: $white;
    }
}
