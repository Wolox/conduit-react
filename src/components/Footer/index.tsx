import { urlWolox } from './constants';
import styles from './styles.module.scss';

function Footer() {
  return (
    <div className={styles.year}>
      Â© {new Date().getFullYear()}{' '}
      <a href={urlWolox} className={styles.link} target="_blank" rel="noreferrer">
        WOLOX
      </a>
    </div>
  );
}

export default Footer;
