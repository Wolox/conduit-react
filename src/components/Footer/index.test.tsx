import { render } from '@testing-library/react';

import Footer from '.';

test('renders component', () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});
