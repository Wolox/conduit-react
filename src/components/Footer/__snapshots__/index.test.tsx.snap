// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component 1`] = `
<DocumentFragment>
  <div
    class="year"
  >
    Â© 2021 
    <a
      class="link"
      href="https://www.wolox.com.ar/"
      rel="noreferrer"
      target="_blank"
    >
      WOLOX
    </a>
  </div>
</DocumentFragment>
`;
